{"name": "Логин с неверными данными", "status": "passed", "steps": [{"name": "Отправляем POST запрос с невалидными данными", "status": "passed", "start": 1746994998374, "stop": 1746994998953}, {"name": "Проверяем код и сообщение об ошибке", "status": "passed", "start": 1746994998953, "stop": 1746994998953}], "parameters": [{"name": "email", "value": "'user@test.com'"}, {"name": "password", "value": "'wrongpass'"}], "start": 1746994998374, "stop": 1746994998954, "uuid": "1095ab2a-4cd3-4c0e-864a-afd1097d4772", "historyId": "edaf95094647d0ca42a19b1df1065bd6", "testCaseId": "fbeae77312463979bb58e896f211518f", "fullName": "tests.test_user_login.TestUserLogin#test_login_with_invalid_credentials", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_user_login"}, {"name": "subSuite", "value": "TestUserLogin"}, {"name": "host", "value": "MacBook-Pro-Filipp-2.local"}, {"name": "thread", "value": "2754-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_user_login"}]}